main.py

```diff
@@ -160,16 +163,16 @@ def setup(self, stage=None):
 
     def _train_dataloader(self):
         return DataLoader(self.datasets["train"], batch_size=self.batch_size,
-                          num_workers=self.num_workers, shuffle=True)
+                          num_workers=self.num_workers, shuffle=True, collate_fn=custom_collate)
```

Comment:
@rromb unfortunately, this was necessary to have `annotations` available during training and sampling. before merging, we should make sure that this does not break any of the other datasets. :)